static/src/javascripts-legacy/projects/common/modules/tailor/tailor.js

```diff
@@ -90,7 +80,25 @@ define([
         });
     }
 
+    /**
+     * Query the user's regular status
+     *
+     * @returns {Promise.<Boolean>}
+     */
+    function isRegular() {
+        return fetchData('suggestions', false).then(function(suggestions) {
+            try {
+                return suggestions.userDataForClient.regular;
+            } catch (e) {
+                return false;
+            }
+        }).catch(function() {
+            return false;
+        });
+    }
+
     return {
-        fetchData: fetchData
+        fetchData: fetchData,
```

Comment:
Out of scope for this, but might be nice to make the fetchData method private, and only expose the functions which extracts the data